gateway/1.6/modules/ROOT/pages/policies-included-external-processing.adoc

```diff
@@ -0,0 +1,149 @@
+= External Processing Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: external processing, processing, flex gateway, gateway, policy
+
+
+[width="100%", cols="5,15"]
+|===
+>s| Policy Name | External Processing
+>s|Summary | Sends the incoming HTTP requests or the outgoing HTTP responses requests to an external gRPC service for additional processing
+>s|Category | Transformation
+>s|First Flex Gateway version available | v1.6.0
+.1+>.^s| Returned Status Codes
+| 403 - IP is rejected
+|===
+
+== Summary
+
+The External Processing policy sends the incoming HTTP request or outgoing HTTP response to an external gRPC service to modify the headers, trailers, body, or any combination of the three. You can also configure this policy to process either the request, response, or both.
+
+Additionally, the external processing service can request different parts of the HTTP request or response after the initial gRPC request has been sent by Flex Gateway to the external processing server. This enables you to send the minimum amount of data necessary. For example, Flex Gateway can initially send only the HTTP requests headers. Then, after some processing, the external processor can request the HTTP request body or trailers from Flex Gateway if necessary. 
+
+The parameters you set for sending the HTTP headers, trailers, and body only configure what is initially sent. The external processing service has access to all parts of the HTTP request or response regardless of what is configured. Configure the parameters to send all the information the processing service needs for every incoming HTTP request or response. For example, if the processing service needs the headers for every request but only needs the body for some of the requests depending on the information found in the header, only configure the parameters to send the headers.
+
+The External Processing policy is an extension of the External Processing filter native to Envoy. To learn more about the External Processing filter, see https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/ext_proc/v3/ext_proc.proto[External Processing Filter^]. Not all External Processing filter parameters are avialible for the External Processing policy. To find what parameters are configurable, see <<configuring-policy-parameters, Configuring Policy Parameters>>. 
+
+NOTE: If other policies modify the HTTP request or response, ensure you order the policies depending on if these modification should be present when the request is sent to the processing service. For more information about ordering policies, see xref:policies-reorder.adoc[].
```

Comment:
```suggestion
NOTE: If other policies modify the HTTP request or response, ensure that you order the policies depending on if these modification should be present when the request is sent to the processing service. For more information about ordering policies, see xref:policies-reorder.adoc[].
```

Reply:
Similar to a comment in the other PR - what does it mean to order policies depending on if modifications should be present? I know you have a more information reference, but it would be good to add more context here.

Reply:
So, you can have multiple policies that each modify an HTTP request.

You may have a policy that injects a Header X into the request. If you want Header X to be included in the request when it is sent to the processing service, you should order the header injection policy to come before the external processing policy. You could also not want it present, and then in this case it should come after or elsewhere.

I think the section as is is fine. It's not really our duty to tell the user how to order their policy, but more how they can order them which is explained in the link doc.

Reply:
I would simplify the sentence, then, and say "If other policies modify the HTTP request or response, ensure that you order the policies accordingly."